### PR TITLE
user login workflow #6

### PR DESCRIPTION
**Pull request by mariza1991**

### CODE REVIEW

File: src/test/java/onlyoffice/com/pages/HomePage.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package onlyoffice.com.pages;
```
Comment (ToshiDono) [inline src/test/java/onlyoffice/com/pages/HomePage.java]: Why pages in `tests` package?

File: src/test/java/onlyoffice/com/pages/MainPage.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import static com.codeborne.selenide.Selenide.$x;
+import static com.codeborne.selenide.Selenide.open;
+import static com.codeborne.selenide.Selenide.page;
+
+public class MainPage {
+
+    private HeaderBlock headerBlock;
+
+    public static MainPage enterToMainPage(){
+        open("http://www.onlyoffice.com/");
```
Comment (ToshiDono) [inline src/test/java/onlyoffice/com/pages/MainPage.java]: `http://www.onlyoffice.com/` magic text. Please extract pages urls in separate class.

File: src/test/java/onlyoffice/com/pages/components/loginPage/LoginFormBlock.java [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package onlyoffice.com.pages.components.loginPage;
+
+public class LoginFormBlock {
+
+    public static final String EMAIL_INPUT = "#txtSignInEmail";
+
```
Comment (ToshiDono) [inline src/test/java/onlyoffice/com/pages/components/loginPage/LoginFormBlock.java]: Extra blank line.

File: src/test/java/onlyoffice/com/pages/components/loginPage/LoginFormBlock.java [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+    public static final String PASSWORD_INPUT = "#txtSignPassword";
+
```
Comment (ToshiDono) [inline src/test/java/onlyoffice/com/pages/components/loginPage/LoginFormBlock.java]: The same.

File: src/test/java/onlyoffice/com/tests/LoginTest.java [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+import org.testng.annotations.Test;
+
+import static onlyoffice.com.pages.MainPage.enterToMainPage;
+
+public class LoginTest extends BaseTest {
+
+    @Test
+    public void loginTest(){
+        enterToMainPage()
```
Comment (ToshiDono) [inline src/test/java/onlyoffice/com/tests/LoginTest.java]: Why we go to Main page but not Login page?

File: src/test/java/onlyoffice/com/tests/LoginTest.java [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+                .clickOnLoginButton()
+                .loginAs("enter@your.email", "password");
```
Comment (ToshiDono) [inline src/test/java/onlyoffice/com/tests/LoginTest.java]: Magic strings. Please extract to separate class or enum or extract to json/yaml file and create reader.

File: src/test/java/onlyoffice/com/pages/components/homePage/HeaderBlock.java [hunk 7]
```diff
@@ -0,0 +1,6 @@
+package onlyoffice.com.pages.components.homePage;
+
+public class HeaderBlock {
+
+    public static final String LOGIN_BTN = "//a[@class='singin menuitem']";
```
Comment (ToshiDono) [inline src/test/java/onlyoffice/com/pages/components/homePage/HeaderBlock.java]: How can I understand that I need to use css or Xpath?

Comment (mariza1991) [reply src/test/java/onlyoffice/com/pages/components/homePage/HeaderBlock.java]: Is it important in the task context?

File: src/test/java/onlyoffice/com/pages/HomePage.java (hunk 1 above)
Comment (mariza1991) [reply src/test/java/onlyoffice/com/pages/HomePage.java]: fixed

File: src/test/java/onlyoffice/com/pages/MainPage.java (hunk 2 above)
Comment (mariza1991) [reply src/test/java/onlyoffice/com/pages/MainPage.java]: fixed

File: src/test/java/onlyoffice/com/pages/components/loginPage/LoginFormBlock.java (hunk 3 above)
Comment (mariza1991) [reply src/test/java/onlyoffice/com/pages/components/loginPage/LoginFormBlock.java]: it is for better visibility

File: src/test/java/onlyoffice/com/tests/LoginTest.java (hunk 5 above)
Comment (mariza1991) [reply src/test/java/onlyoffice/com/tests/LoginTest.java]: realized in LoginStep, fixed

File: src/test/java/onlyoffice/com/pages/components/loginPage/LoginFormBlock.java [hunk 8]
```diff
@@ -0,0 +1,10 @@
+package onlyoffice.com.pages.components.loginPage;
+
+public class LoginFormBlock {
+
+    public static final String EMAIL_INPUT = "#txtSignInEmail";
```
[the remark this answers is not in the document]
Comment (mariza1991) [reply src/test/java/onlyoffice/com/pages/components/loginPage/LoginFormBlock.java]: Is it important in the task context?

**Pull request merged**